source/full_shutdown.rst

```diff
@@ -0,0 +1,205 @@
+.. include:: vars.rst
+
+=======================
+Full Shutdown Procedure
+=======================
+
+In case a full shutdown of the system is required, we advise to use the
+following order:
+
+* Perform a graceful shutdown of all virtual machine instances
+* Stop Ceph (if applicable)
+* Put all nodes into maintenance mode in Bifrost
+* Shut down compute nodes
```

Comment:
nit: this lists shutting down different types of nodes separately, but the procedure only stops the services separately, then shuts down all nodes at once.